### ISSUE TITLE
False positive in isPeekNextLineComment for indented lines

### ISSUE THREAD

**Issue by stasm** [projectfluent/fluent.js#251]
`isPeekNextLineComment` gives a false positive when the next line doesn't start with `#` but is indented by at least `(n + 1)` spaces, where `n` is the current comment level.

The following FTL:

```properties
# Comment
  junk
```

…is parsed as:

```json
{
  "type": "Resource",
  "body": [
    {
      "type": "Comment",
      "annotations": [],
      "content": "Comment\n junk\n"
    }
  ]
}
```

**Comment by stasm** [comment]
Hey @zbraniecki, do you have an idea what's causing this?

**Comment by stasm** [comment]
Fixed on the `feature/syntax-0.6` branch by #252. #253 tracks merging the whole branch into master.
